### PR TITLE
Add a Strongswan <-> Cloud VPN interop guide

### PR DESCRIPTION
**Pull request by Civil**
This commit adds an interoperability guide between strongswan and
Cloud VPN. Guide covers Polcy-based VPN and Dynamic (BGP).

It was tested to work on a test project that simulates On-Prem with
Strongswan, but based on GCP.

### CODE REVIEW

**Comment by ToddKopriva** [comment]
I'll do a production pass next. 

@Civil and @PTDelaney : Our build process only allows for one author in the frontmatter. If you want to acknowledge two authors, you can do so in the body of the article, as is done in this tutorial:
https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/cloud-iot-gateways-rpi/index.md

File: tutorials/using-cloud-vpn-with-strongswan/index.md [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+
+## Prerequisites
+
+To use a strongSwan with Cloud VPN make sure the following prerequisites have been met:
+
+*   VM or Server that runs strongSwan is healthy and has no known issues. 
+*   There is root access to the strongSwan instance.
+*   Your on-premises firewall allows UDP port 500, UDP port 4500, and ESP packets.
+*   You should be able to configure your on-premises router to route traffic through
+    Strongswan VPN gateway. Some environments might not give you that option.
```
Comment (ToddKopriva) [inline tutorials/using-cloud-vpn-with-strongswan/index.md]: The capitalization of 'strongSwan' seems somewhat inconsistent.

**Comment by jschneider11** [comment]
Can you please add @ashishverm as a reviewer? I don't have that capability. Thanks!

File: tutorials/using-cloud-vpn-with-strongswan/index.md (hunk 1 above)
Comment (Civil) [reply tutorials/using-cloud-vpn-with-strongswan/index.md]: Should be consistent now. I've also fixed the same issue for "BIRD" (Bird -> BIRD).

**Comment by PTDelaney** [comment]
The company is called strongSwan. There is a strongswan file that you run
to install the app. There should not be any instances of Strongswan.

-- 
Peter Delaney | Technical Editor | Google Cloud Platform

**Comment by Civil** [comment]
Last commit should change it to consistently mention "strongswan" as "strongSwan" in the text. Only exception is config files, filesystem paths and service name, where it's called "strongswan" (by Debian's policy).

```
$ grep -c 'Strongswan' index.md 
0
$ grep -c 'strongSwan' index.md 
39
```
I've also done the same for BIRD:
```
$ grep -c 'Bird' index.md 
0
$ grep -c 'BIRD' index.md 
7
```

So I think it's ready for the next rounds of reviews.

**Comment by ToddKopriva** [comment]
Thanks. I'm going to hold off on the production pass until you've updated the author entry in the frontmatter and until I've heard back from Ashish (on an internal email thread) about whether he wants to do a review.

**Comment by Civil** [comment]
Author was updated in my last commit. It now mentions only me and as you've adviced I mentioned him on a next line.

**Comment by ToddKopriva** [comment]
Thanks. I'll continue with the production pass, then.

**Comment by ToddKopriva** [comment]
published:
https://cloud.google.com/community/tutorials/using-cloud-vpn-with-strongswan

**Pull request merged**